client/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  entry: './main.js',
+  output: {
+    filename: 'out.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        loader: 'babel-loader',
```

Comment:
this uses `.babelrc` by default. Alternatively one could remove the `.babelrc` and just inline the babel config here.